examples/website/plot/plot-layer/axes-layer.ts

```diff
@@ -104,15 +116,7 @@ export default class AxesLayer extends Layer {
       instanceIsTitle: {size: 1, update: this.calculateInstanceIsTitle, noAlloc: true}
     });
 
-    this.setState(
-      Object.assign(
-        {
-          numInstances: 0,
-          labels: null
-        },
-        this._getModels(gl)
-      )
-    );
+    this.setState(Object.assign({instanceCount: 0}, this._getModels(gl)));
```

Comment:
I'll rename this back to `numInstances` - hadn't realized that `Layer.ts` relies on this naming.